ui/src/App/index.jsx

```diff
@@ -68,13 +68,13 @@ export default class App extends Component {
   });
 
   httpLink = new HttpLink({
-    uri: process.env.GRAPHQL_ENDPOINT,
+    uri: window.env.GRAPHQL_ENDPOINT,
```

Comment:
I cannot google this (I just keep getting results about Windows the OS) - what exactly is `window.env`? Do you have any links to read on it?

Reply:
`window` is the global variable available to the browser: https://developer.mozilla.org/en-US/docs/Web/API/Window.

Reply:
oic. I started to suspect that was something custom - that explains why I had trouble finding it :)